docs/api.md

```diff
@@ -0,0 +1,366 @@
+      "name": "modelname:tags",
+      "modified_at": "2023-08-04T08:52:19.385406455-07:00",
+      "size": size
+    }
+  ]
+}
+```
+
+### Example
+
+#### Request
+
+```shell
+curl --location --request GET 'http://localhost:11434/api/tags'
+```
+
+#### Response
+
+```JSON
+{
+    "models": [
+        {
+            "name": "llama2:70b",
+            "modified_at": "2023-08-04T08:52:19.385406455-07:00",
+            "size": 38871966966
+        },
+        {
+            "name": "llama2:70b-chat-q4_0",
+            "modified_at": "2023-08-04T09:21:27.703371485-07:00",
+            "size": 38871974480
+        },
+        {
+            "name": "midjourney-prompter:latest",
+            "modified_at": "2023-08-04T08:45:46.399609053-07:00",
+            "size": 7323311708
+        },
+        {
+            "name": "raycast_orca:3b",
+            "modified_at": "2023-08-04T06:23:20.10832636-07:00",
+            "size": 1928446602
+        },
+        {
+            "name": "stablebeluga:13b-q4_K_M",
+            "modified_at": "2023-08-04T09:48:26.416547463-07:00",
+            "size": 7865679045
+        }
+    ]
+}
+```
+
+## Copy a Model
+
+**POST /api/copy**
+
+### Description
+
+**Copy** will copy a model from one name to another. This is useful for creating a new model from an existing model. It is often used as the first step to renaming a model.
+
+### Request
+
+The **Copy** endpoint takes a JSON object with the following fields:
+
+```JSON
+{
+  "from": "modelname",
+  "to": "newmodelname"
+}
+```
+
+### Response
+
+There is no response other than a 200 status code.
+
+### Example
+
+#### Request
+
+```shell
+curl --location --request POST 'http://localhost:11434/api/copy' \
+--header 'Content-Type: text/plain' \
+--data-raw '{
+    "source": "MyCoolModel",
+    "destination": "ADifferentModel"
+}'
+```
+
+#### Response
+
+No response is returned other than a 200 status code.
+
+## Delete a Model
+
+**DEL /api/delete**
+
+### Description
+
+**Delete** will delete a model from the local machine. This is useful for cleaning up models that are no longer needed.
+
+### Request
+
+The **Delete** endpoint takes a JSON object with a single key/value pair for modelname. For example:
+
+```JSON
+{
+  "model": "modelname"
+}
+```
+
+### Response
+
+No response is returned other than a 200 status code.
+
+### Example
+
+#### Request
+
+```shell
+curl --location --request DELETE 'http://localhost:11434/api/delete' \
+--header 'Content-Type: text/plain' \
+--data-raw '{
+    "name": "adifferentModel"
+}'
+```
+
+#### Response
+
+No response is returned other than a 200 status code.
+
+## Pull a Model
+
+**POST /api/pull**
+
+### Description
+
+**Pull** will pull a model from a remote registry. This is useful for getting a model from the Ollama registry and in the future from alternate registries.
+
+### Request
+
+The **Pull** endpoint takes a JSON object with the following fields:
+
+```JSON
+{
+  "name": "modelname",
+  "registry": "registryname"
```

Comment:
This isn't an option

Reply:
weird, i didn't think i put that in there. I was thinking it...